challenge_1.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
Reminder, if you have nested HTML tags try to indent them such as you would code. It makes it easier to read.